outline.md

```diff
@@ -0,0 +1,292 @@
+# Outline for Pulsar documentation
+
+This uses the og atom Flight manual as a base, and converts and refactors it for modern standards. This will be converted to and tracked from our projects using seperate ones for each section.
+
+## Old Already "Finished" sections
+
+These are alredy in our atom-archive folder. These just need to be reformatted, branding changed, and use ours and more modern standards. Especially since as we move to more updated libraries methods mentioned in these sections will change and/or break.
+
+### Chapter 1: Getting Started
+
+- Why Atom?
+- Installing Atom
+- Atom Basics
+- Summary
+
+### Chapter 2: Using Atom
+
+- Atom Packages
+- Moving in Atom
+- Atom Selections
+- Editing and Deleting Text
+- Find and Replace
+- Snippets
+- Autocomplete
+- Folding
+- Panes
+- Pending Pane Items
+- Grammar
+- Version Control in Atom
+- GitHub package
+- Writing in Atom
+- Basic Customization
+- Summary
+
+### Chapter 3: Hacking Atom
+
+- Tools of the Trade
+- The Init File
+- Package: Word Count
+- Package: Modifying Text
+- Package: Active Editor Info
+- Creating a Theme
+- Creating a Grammar
+- Creating a Legacy TextMate Grammar
+- Publishing
+- Iconography
+- Debugging
+- Writing specs
+- Handling URIs
+- Cross-Platform Compatibility
+- Converting from TextMate
+- Hacking on Atom Core
+- Contributing to Official Atom Packages
+- Creating a Fork of a Core Package in atom/atom
+- Maintaining a Fork of a Core Package in atom/atom
+- Summary
+
+### Chapter 4: Behind Atom
+
+- Configuration API
+- Keymaps In-Depth
+- Scoped Settings, Scopes and Scope Descriptors
+- Serialization in Atom
+- Developing Node Modules
+- Interacting With Other Packages Via Services
+- Maintaining Your Packages
+- How Atom Uses Chromium Snapshots
+- Summary
+
+### Reference: API
+
+**_ this section is not loaded in since it's in JSON files we need to parse. _**
```

Comment:
```suggestion
**_this section is not loaded in since it's in JSON files we need to parse._**
```

Reply:
This one probably isn't making it into the final file. I'm hoping before that point we've reorganized, especially the sections they weren't sure where to put them. But I'll add it for now...